src/main/webapp/maps.html

```diff
@@ -18,12 +18,12 @@
       padding: 10px;
     }
   </style>
-  <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBxd60VBUhGDGcDgEEwgyFkEfVtcG6xiZU"></script>
+  <script src="/js/markerclusterer.js"></script>
```

Comment:
Nit pick - I think these are usually ordered in alphabetical order, so maybe you could switch this with the next one, since maps is before marker.

Reply:
Really inconsequential detail haha

Reply:
Oh its ordered that way because sometimes one js script has to be called first before the other. But I'd be removing this line because its not fully implemented yet :)